### PR TITLE
Define process for requesting TC decision, clarify TC voting process

### PR DESCRIPTION
**Pull request by jack-berg**
Attempting to capture the discussion in the 6/14/23 TC meeting. 

cc @open-telemetry/technical-committee.

### CODE REVIEW

File: tech-committee-charter.md
Comment (reyang) [inline tech-committee-charter.md:107]: Are we okay with any participants or we want to scope this to project leads for now? I'm considering the situation where the SIG reached consensus among majority of folks while a single participant wanted to escalate to the TC.

**Review by reyang — APPROVED** [review]
LGTM with a non-blocking comment.

File: tech-committee-charter.md
Comment (yurishkuro) [inline tech-committee-charter.md:103]: > a simple majority of all TC members

I think this is a bit ambiguous: majority of all TC, or majority of those who voted? If the latter, then there should be a quorum requirement.

File: tech-committee-charter.md
Comment (jack-berg) [reply tech-committee-charter.md:107]: I think minimally we should scope it to otel community members. Further narrowing to project leads seems helpful if there's a simple definition for what a project lead is. Could say that it has to be 1 or 2 maintainers / approvers.

**Review by tigrannajaryan — APPROVED** [review]

File: tech-committee-charter.md
Comment (arminru) [reply tech-committee-charter.md:103]: Agreed!

**Review by carlosalberto — APPROVED** [review]

File: tech-committee-charter.md
Comment (yurishkuro) [inline tech-committee-charter.md:103]: > members who abstain do not count towards the quorum

What is the motivation for this? in the TC discussion the feeling was that maybe abstaining should still count as a form of a vote and be part of the quorum.

File: tech-committee-charter.md
Comment (yurishkuro) [inline tech-committee-charter.md:109]: Do we want to specify a process for TC to decline to vote on an issue? For example, SCOTUS requires 4 out of 9 justices to vote yes to accept a case. I don't think TC should be obligated to take any conceivable disagreement that might arise in some SIG.

File: tech-committee-charter.md
Comment (jack-berg) [reply tech-committee-charter.md]: That was just me misinterpreting the quorum [suggestion here](https://github.com/open-telemetry/community/pull/1547#discussion_r1230234126). I've pushed a commit which counts abstaining towards quorum.

File: tech-committee-charter.md
Comment (jack-berg) [reply tech-committee-charter.md:109]: I'm open to this but don't want to be over specify the process. I think the "A TC member should review the summary and may request additional details or other changes to more accurately frame the issue" leaves room for the TC to reject the request, but maybe you could suggest some phrasing that makes that more explicit?

File: tech-committee-charter.md
Comment (jack-berg) [reply tech-committee-charter.md]: I pushed a commit which counts abstaining towards the quorum requirement. This means we could scenarios like:

1. 4 members abstain, 1 member votes for option a: option a wins
2. 3 members abstain, 1 member votes for option a, 1 member votes for option b: the vote is not resolved until somebody breaks the tie by voting instead of abstaining, or an additional member casts their vote in favor of one of the options

So in scenario 2 there is still the possibility for an indeterminant vote, but I think that will be rare enough that we can work it out among ourselves.

**Review by yurishkuro — APPROVED** [review]

**Comment by jack-berg** [comment]
@open-telemetry/governance-committee this is ready to merge.

**Pull request merged**
